.idea/deploymentTargetDropDown.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we've added the `.idea/` to gitignore ? Why is this showing up again, could you please check?

Reply:
```.idea``` folder is added in gitignore 
![image](https://user-images.githubusercontent.com/55796198/129935011-9d4002a1-06ac-409d-9169-2d93add5cacd.png)

Reply:
what about ?
`**/.idea/*`